cmd/cluster/azure/create.go

```diff
@@ -67,6 +69,8 @@ func bindCoreOptions(opts *RawCreateOptions, flags *pflag.FlagSet) {
 	flags.StringVar(&opts.NetworkSecurityGroupID, "network-security-group-id", opts.NetworkSecurityGroupID, "The Network Security Group ID to use in the default NodePool.")
 	flags.StringToStringVarP(&opts.ResourceGroupTags, "resource-group-tags", "t", opts.ResourceGroupTags, "Additional tags to apply to the resource group created (e.g. 'key1=value1,key2=value2')")
 	flags.StringVar(&opts.SubnetID, "subnet-id", opts.SubnetID, "The subnet ID where the VMs will be placed.")
+	flags.StringVar(&opts.IssuerURL, "oidc-issuer-url", "", "The OIDC provider issuer URL")
+	flags.StringVar(&opts.ServiceAccountTokenIssuerKeyPath, "sa-token-issuer-private-key-path", "", "The path to the private key for the service account token issuer")
```

Comment:
These flags are already in the docs :stuck_out_tongue: 

https://github.com/openshift/hypershift/blob/ed8474179db0887e87f7f6f4b4f60a150503431b/contrib/managed-azure/Setting%20up%20workload%20identity%20for%20Azure%20HC.md?plain=1#L260-L261

Reply:
I was surprised we didn't have this option/flag for AWS already, but it turns out we do manual manipulation of the HC spec in e2e

https://github.com/openshift/hypershift/blob/ed8474179db0887e87f7f6f4b4f60a150503431b/test/e2e/util/hypershift_framework.go#L243-L245

Reply:
can we do `s/path/file/`? I read path as directory and for the other flags we use `file`.

Reply:
To be fair, those docs say you need this PR in order to run them 😅